modules/_labs/select/react/stories/stories.tsx

```diff
@@ -322,4 +326,10 @@ storiesOf('Labs|Select/React/Visual Testing', module)
         )}
       </ComponentStatesTable>
     </StaticStates>
-  ));
+  ))
+  .addParameters({
+    canvasProviderDecorator: {
+      theme: customColorTheme,
+    },
+  })
+  .add('Theming', () => <SelectStates />);
```

Comment:
I feel like there should be Theming states tables for `States (Menu On)` and `States (Options)` as well, but I'm leaning towards handling them later given both tables are being modified considerably in other PRs (`States (Menu On)` in #641, and `States (Options)` in #606).

Reply:
I went ahead and added the states tables. We'll resolve conflicts later.